README.md

```diff
@@ -4,10 +4,71 @@ A [Work Item](https://privacycg.github.io/charter.html#work-items)
+
+## Additional keying
+
+[whatwg/html #4966](https://github.com/whatwg/html/pull/4966) defined the [top-level origin](https://html.spec.whatwg.org/multipage/webappapis.html#top-level-origin) concept for an environment and HTML also defines site and [obtain a site](https://html.spec.whatwg.org/multipage/origin.html#obtain-a-site). Together these allow for a definition of top-level site, which most user agents are targeting as additional key.
+
+For some user agent state it might be beneficial to add even more keys, e.g., to prevent attacks between framed documents. [shivanigithub/http-cache-partitioning #2](https://github.com/shivanigithub/http-cache-partitioning/issues/2) has some relevant discussion.
+
+## Relaxing additional keying
+
+For some user agent state (Cookies and Storage below in particular are under discussion) the additional keying might be relaxable. This is tracked by [The Storage Access API](https://privacycg.github.io/storage-access/) though also necessarily has to inform the work noted below as it might impact architecture decisions.
+
+## Blocking
+
+Aside from using additional keying, outright blocking of the user agent state is also considered at times, e.g., for Cookies or as happens today for Storage in opaque origins. This is not likely to be web compatible nor even desirable for all user agent state, but could well be a valid solution for some.
+
+## User agent state
+
+An enumeration of user agent state and ongoing standards activity.
+
+* Cookies
+* Network state:
+   * HTTP cache ([whatwg/fetch #943](https://github.com/whatwg/fetch/pull/943))
+   * Connection pooling
+   * DNS
+   * HTTP authentication
+   * Alt-Svc
+   * Fonts
+   * HSTS
+   * TLS client certificates
+   * TLS session identifiers
+   * Prefetch
+   * Preconnect
+* Storage ([whatwg/storage #18](https://github.com/whatwg/storage/issues/18)):
+   * Indexed DB
+   * Cache API
+   * `localStorage`
+   * `sessionStorage`
+   * `history.pushState()`
+   * `new Notification()`
+   * AppCache (actively being removed, probably not relevant)
+* Storage (communication channels):
+   * `BroadcastChannel`
+   * Shared workers
+   * Service workers
+   * Web Locks
+* Web Authentication
+* WebRTC’s `deviceId` ([w3c/mediacapture-main #675](https://github.com/w3c/mediacapture-main/issues/675))
+* `window.name`
+* Browsing context group's agent cluster map (only observable with popups)
+* Permissions ([Feature Policy](https://w3c.github.io/webappsec-feature-policy/) allows these to be disabled by default when the top-level site is not equal to the current site and require explicit delegation in such cases)
+* Credentials (username and password storage; not a standardized feature)
+
+## Acknowledgments
+
+The author of this document was inspired by Chromium’s NIK, Firefox’s FPI/dFPI/StoragePrincipal, Safari’s ITP, XS-Leaks, and the many people wanting to improve these aspects of the web.
```

Comment:
May be mention Chromium's Network Isolation Key (as NIK might not be clear to those who are not familiar with it.)